README.md

```diff
@@ -561,6 +561,59 @@ Use it to type in your key and install.
 
 Go back to Step 4A-4 and then finish with Step 4A-5.
 
+---
+## Forks
+
+### Which Fork Should I Use?
+
+If you have to ask, you're not ready for forks.
+
+Begin your research in Discord #custom-forks
+
+Please do not ask about forks outside of that channel.
+
+If you're new, please start with `commaai/master-ci` and use it for two weeks. This is the latest official version with only lateral support. This will give you a good baseline to compare the other forks to.
+
+> [!CAUTION]
+> Some forks will brick your C3X. \
+> Most forks are not made for C3X because C3X is less than a year old.
+> 
+> Some forks contain banned code. \
+> Using it will get you banned by the comma.ai company.
+> 
+> Some forks have nudgeless-lane-change. \
+> Simply clicking the turn signal will move your car to the next lane. \
+> Without any checks. \
+> Yes, it will drive into the car next to you.
+> 
+> Some forks play a blood-curdling goat scream at max volume randomly.
+> 
+> The danger in using forks is not theoretical because it has full control over your car.
```

Comment:
I think maybe you should add that people should get familiar with the communities, especially their Discord communities and presence before using a fork. There may be strange changes and other errata that they might want to report. Features get a lot less testing and their interaction with SecOC/TSK vehicles may be suspect.

Reply:
That's a good point

Reply:
On a second thought, yes to getting familiar with their Discord (maybe we should put a link on the table).

As for the SecOC interaction, I think driving options are separate from SecOC. It's not like there's MADS and then a separate MADS for SecOC.